src/frontend/src/lib/utils/transactions.utils.ts

```diff
@@ -40,8 +61,36 @@ export const mapAllTransactionsUi = ({
 		}
 
 		if (isNetworkIdEthereum(networkId)) {
-			// TODO: Implement Ethereum transactions
-			return acc;
+			// TODO: remove Sepolia transactions when the feature is complete; for now we use it for testing
```

Comment:
Can you move those initialization outside of the reduce? It's what I meant but, maybe that's not possible?

Reply:
ah yes, i can, but only of two ways:

1) we initialize always, independently of having Eth/Sepolia tokens

OR

2) I initialize only if we have Eth/Sepolia tokens outside the reduce, but that means that i have to "scan" the token list twice, with `find` first and then in reduce

Which one is preferable? I am more on the first one eheh

Reply:
I tend for the 1., not the 2. 
But if you rather like to keep it in the reduce after all, I can also understand now.

Reply:
changed to do the 1! I prefer it too